bankingAsssignment'js

```diff
@@ -0,0 +1,183 @@
+class Transaction {
+  constructor(type, amount) {
+    this._type = type;
+    this._amount = amount;
+    this._timestamp = new Date();
+  }
+
+  get type() {
+    return this._type;
+  }
+
+  set type(value) {
+    this._type = value;
+  }
+
+  get amount() {
+    return this._amount;
+  }
+
+  set amount(value) {
+    this._amount = value;
+  }
+
+  get timestamp() {
+    return this._timestamp;
+  }
+
+  set timestamp(value) {
+    this._timestamp = value;
+  }
+}
+
+class BankAccount {
+  constructor() {
+    this._accountNumber = Math.floor(Math.random() * 1000000000);
+    this._firstName = '';
+    this._lastName = '';
+    this._accountHolder = '';
+    this._balance = 0;
+    this._transactions = [];
+    this.transaction_amounts = [];
+  }
+
+  get accountNumber() {
+    return this._accountNumber;
+  }
+
+  set accountNumber(value) {
+    this._accountNumber = value;
+  }
+
+  get firstName() {
+    return this._firstName;
+  }
+
+  set firstName(value) {
+    this._firstName = value;
+  }
+
+  get lastName() {
+    return this._lastName;
+  }
+
+  set lastName(value) {
+    this._lastName = value;
+  }
+
+  get accountHolder() {
+    return this._accountHolder;
+  }
+
+  set accountHolder(value) {
+    this._accountHolder = value;
+  }
+
+  get balance() {
+    return this._balance;
+  }
+
+  set balance(value) {
+    this._balance = value;
+  }
+
+  get transactions() {
+    return this._transactions;
+  }
+
+  set transactions(value) {
+    this._transactions = value;
+  }
+ get transaction_limit(){
```

Comment:
Well done, but this is not how daily limit works, its a feature to prevent the user from withdrawing more than he/she is allowed for a day